tests/README.md

```diff
@@ -59,6 +60,24 @@ pip install -U -r requirements.txt
 deactivate
 ```
 
+## Fix issue with podman pasta networking provider
```

Comment:
Since we have already a section for configuring podman (for the sockets), maybe we can group these two? For example:
```
## Configuring podman
### Enable podman socket access for users
### Podman network provider
```

Reply:
Done